main.tf

```diff
@@ -13,6 +13,7 @@ resource "aws_cloudfront_origin_access_identity" "this" {
 # ---------------------------------------------------------------------------------------------------------------------
 
 resource "aws_s3_bucket_policy" "this" {
+  count  = "${var.enable_bucket_policy == true ? 1 : 0}"
```

Comment:
Not sure whether we should write it for terraform 0.13 onwards without the quotes

Reply:
@hingstarne judging by the lines below, that would be the case (not necessary to support 0.11 syntax as the module isn't ready as a whole)
Ref: https://github.com/freeletics/terraform-aws-cloudfront-s3-origin-acm/blob/feature/enable_policy/main.tf#L17-L18